dpcpp/matrix/coo_kernels.dp.cpp

```diff
@@ -61,11 +70,311 @@ namespace dpcpp {
 namespace coo {
 
 
+constexpr int default_block_size = 256;
+constexpr int warps_in_block = 4;
```

Comment:
Is this correct? Don't you need 7 or 8 warps/subgroups (number of EU per subslice), * 2 (two ALUs per EU) each of size 16 or whichever?
See https://software.intel.com/content/www/us/en/develop/download/architecture-overview-for-intel-processor-graphics-gen11.html

Reply:
I think 7 or 8 should be used in oversubscription?